actpool/actioniterator/actioniterator.go

```diff
@@ -86,3 +82,26 @@ func (ai *actionIterator) LoadNextAction() {
 		heap.Pop(&ai.heads)
 	}
 }
+
+// Next load next action of account of top action
+func (ai *actionIterator) Next() action.Action {
+	if len(ai.heads) == 0 {
```

Comment:
for len(ai.heads) > 0 {

        headAction := ai.heads[0]

        err := ai.validator.Validate(bestAction)

        if err == nil {

            return headAction

        }

        switch err {

            ...

        }

     }

     return nil